pkg/minikube/cluster/cluster.go

```diff
@@ -315,3 +316,40 @@ func CreateSSHShell(api libmachine.API, args []string) error {
 	}
 	return client.Shell(strings.Join(args, " "))
 }
+
+func GetDashboardURL(api libmachine.API) (string, error) {
+	host, err := checkIfApiExistsAndLoad(api)
+	if err != nil {
+		return "", err
+	}
+
+	ip, err := host.Driver.GetIP()
+	if err != nil {
+		return "", err
+	}
+
+	port, err := getDashBoardPort()
+	if err != nil {
+		return "", err
+	}
+
+	return fmt.Sprintf("http://%s:%s", ip, port), nil
+}
+
+func getDashBoardPort() (string, error) {
+	//$(kubectl get service kubernetes-dashboard --namespace=kube-system -o=jsonpath='{.spec.ports[0].nodePort}{"\n"}')
```

Comment:
We should probably try to use the kubernetes API directly instead of shelling out to kubectl. You'll want to use this:
https://github.com/kubernetes/kubernetes/blob/dae5ac482861382e18b1e7b2943b1b7f333c6a2a/pkg/client/unversioned/services.go#L33

You'll need something like this: https://github.com/kubernetes/kubernetes/blob/dae5ac482861382e18b1e7b2943b1b7f333c6a2a/pkg/client/unversioned/client.go#L77

To get the ServiceInterface object. Here's a rough example:
http://stackoverflow.com/questions/32554893/how-can-i-create-a-simple-client-app-with-the-kubernetes-go-library

You'll have to change the auth a bit in the client contructor though.